pushapkscript/docker.d/init_worker.sh

```diff
@@ -44,39 +44,34 @@ case $COT_PRODUCT in
     case $ENV in
       dev|fake-prod)
 
-        echo "dummy" > $CONFIG_DIR/fake_cert.p12
-        export GOOGLE_CREDENTIALS_FENIX_DEP_PATH=$CONFIG_DIR/fake_cert.p12
-        export GOOGLE_CREDENTIALS_FOCUS_DEP_PATH=$CONFIG_DIR/fake_cert.p12
-        export GOOGLE_CREDENTIALS_REFERENCE_BROWSER_DEP_PATH=$CONFIG_DIR/fake_cert.p12
+        echo "dummy" > $CONFIG_DIR/fake_cert.json
```

Comment:
I suppose we could make fake_cert.json more realistic...not sure that gives us much, and certainly not a priority.